apps/store/controllers/assets-router.jag

```diff
@@ -40,8 +41,10 @@ var mapper = function (path) {
 
 var handleApiEndpoint = function (request, session, uriParams) {
     var type = uriParams.type;
-    if(!validateType(type)){
-        response.sendError(404, 'Unabled to locate a active endpoint for the type: ' + type);
+    var tenantId = getTenantId(request, session);
+    if(!validateType(type, tenantId)){
```

Comment:
Thanks @daneshk , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/842/commits/d825d8c011cf7d57ddf865fadb74847d5f3fa9fd)